packages/juce-blueprint/tsconfig.json

```diff
@@ -1,11 +1,19 @@
 {
   "compilerOptions": {
+    "module": "commonjs",
     "target": "es6",
+    "lib": ["es2015", "dom"],
     "declaration": true,
-    "module": "commonjs",
+    "sourceMap": true,
+    "isolatedModules":true,
+    "esModuleInterop": true,
     "allowJs": true,
+    "noImplicitAny": false,
     "strict": true,
-    "esModuleInterop": true
+    "strictNullChecks": true,
+    "skipLibCheck": true,
+    "jsx": "react"
   },
-  "include": ["./esm/**/*"]
+  "include": ["./src/**/*"],
+  "exclude": ["./src/lib/polyfill.js"]
```

Comment:
Linters in various IDE's complained about this file without excluding polyfill.js. Maybe a .ts extension in the include pattern is better?